storage/simple_tree.hpp

```diff
@@ -60,12 +61,12 @@ class SimpleTree
     return Value() < other.Value();
   }
 
-  /// sorts siblings independently on each level by default
-  void Sort(bool onlySiblings = false)
+  /// sorts children independently on each level by default
+  void Sort(bool onlyChildren = false)
```

Comment:
I removed this flag at all. Now the methods is used only in tests anyway. I'm not in a hurry to remove all unused methods of the class. I'll probably will need them while working on the downloader. But I left an appropriate TODO in the beginning of the header.